conda-store-server/conda_store_server/server/auth.py

```diff
@@ -436,7 +436,9 @@ async def post_login_method(
             samesite="strict",
             domain=self.cookie_domain,
             # set cookie to expire at same time as jwt
-            max_age=(authentication_token.exp - datetime.datetime.utcnow()).seconds,
+            max_age=int(
+                (authentication_token.exp - datetime.datetime.utcnow()).total_seconds()
+            ),
```

Comment:
See https://github.com/conda-incubator/conda-store/pull/559/files#r1364084772 for context on this.

Reply:
A fun bug :smile: glad we got to the bottom of this one.